docs/user/metrics/datetime.md

```diff
@@ -0,0 +1,76 @@
+# Datetimes
+
+**STATUS: Not implemented.**
+
+Datetimes are used to record an absolute date and time, for example the date and
+time that the application was first run.
+
+The device's offset from UTC is recorded and sent with the datetime value in the
+ping.
+
+To measure the distribution of multiple timespans, see [Timing
+Distributions](timing_distribution.md). To record absolute times, see
+[Datetimes](datetime.md).
+
+## Configuration
+
+Datetimes have a required `time_unit` parameter to specify the smallest unit
+of resolution that the timespan will record. The allowed values for `time_unit` are:
+
+   `nanosecond`, `microsecond`, `millisecond`, `second`, `minute`, `hour`, and
```

Comment:
I know this is just copied over but: would a list with one item per line more readable?